frontend/javascripts/admin/project/project_list_view.js

```diff
@@ -48,6 +51,7 @@ type State = {
   searchQuery: string,
   isTransferTasksVisible: boolean,
   selectedProject: ?APIProjectWithAssignments,
+  taskTypeName: string,
```

Comment:
```suggestion
  taskTypeName: ?string,
```

since the name can be null